x/amm/spec/06_swap_txs_reordering.md

```diff
@@ -0,0 +1,35 @@
+# Swap transactions reordering
+
+To provide lowest slippage as possible on the block, module sort the swap requests into a specific order.
+
+This removes possibility of MEV attack on swap transactions.
+
+Instead of executing messages directly, on the endblocker, we store the requests on transient store and execute in specific order to minimize maximum slippage.
```

Comment:
@jelysn How is the transient store structured to ensure efficient access and modification of swap requests?

Reply:
It will be put as in this format
`prefix-denom-pool-denom-pool...`